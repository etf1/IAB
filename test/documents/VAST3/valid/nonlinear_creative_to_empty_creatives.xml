<!-- <CompanionAds> and <NonLinearAds> are ignored but should be parsed/validated -->
<VAST version="3.0">
    <Ad id="601364">
        <InLine>
            <AdSystem>Acudeo Compatible</AdSystem>
            <AdTitle>VAST 2.0 Instream Test 1</AdTitle>
            <Description>VAST 2.0 Instream Test 1</Description>
            <Error>http://myErrorURL/error</Error>

            <Impression>http://myTrackingURL/impression</Impression>
            <Creatives>
                <Creative AdID="601364">
                    <NonLinearAds></NonLinearAds>
                </Creative>
            </Creatives>
        </InLine>
    </Ad>
</VAST>
