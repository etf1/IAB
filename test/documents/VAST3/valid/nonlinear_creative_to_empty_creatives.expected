{
  "version": "3.0",
  "ads": [
    {
      "adType": 1,
      "adTitle": "VAST 2.0 Instream Test 1",
      "description": "VAST 2.0 Instream Test 1",
      "adSystem": {
        "name": "Acudeo Compatible"
      },
      "id": "601364",
      "error": "http://myErrorURL/error",
      "impressions": [
        {
          "uri": "http://myTrackingURL/impression"
        }
      ],
      "creatives": []
    }
  ]
}
